File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/risorse/RisorsaMacchina.ts
import { ParcoMacchine } from '../../anagrafiche/ParcoMacchine';
import { RisorsaTimeSheet } from './RisorsaTimeSheet';

export class RisorsaMacchina extends RisorsaTimeSheet {
    macchina: ParcoMacchine;

    constructor() {
        super();
        this.classType = 'RisorsaMacchina';
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/risorse/RisorsaMacchina.cs
using AgronicaCoreModelsSTD.anagrafiche;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.risorse
{
    public class RisorsaMacchina: RisorsaTimeSheet
    {
        /**
         * Puntatore ad Anagrafica della macchina
         */
        public ParcoMacchine macchina { get; set; }

        public RisorsaMacchina()
        {
            classType = costanti.ClassType.RisorsaMacchina;
        }

    }
}
